tests/manual/test-procedure.rst

```diff
@@ -248,6 +248,92 @@ Code block 1::
     from microbit import *
     display.show(Image.PACMAN)
 
+Test Case: module.py file can be loaded by file picker and used in main.py
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- Load by Load/Save > Add file the file ``emoji.py``
+- [ ] Confirm the file shows up in the files list with the same title.
+- Return to the editor and replace the current script with the following:
+
+```
```

Comment:
This is restructured text file, so code blocks needs to be formatted slightly differently, have a look at the previous test case for an example.

![image](https://user-images.githubusercontent.com/29712657/61792909-bd00c000-ae15-11e9-862c-93f1b79efde0.png)

Reply:
fixed